src/TextBubbleSkin.js

```diff
@@ -241,6 +241,11 @@ class TextBubbleSkin extends Skin {
         this._renderedScale = scale;
     }
 
+    updateSilhouette (scale = [100, 100]) {
```

Comment:
Why was this needed?

Reply:
This ensures that text bubbles' "silhouettes" behave consistently on the CPU render path. Since text bubbles are no longer rendered on the CPU path, this is technically not necessary but should prevent future weirdness like what happened with the test here where text bubbles appeared to not count as being touched for the first frame.